### ISSUE TITLE
Add CL arg to ignore inability to exclusively lock Journal directory

### ISSUE THREAD

**Issue by MalicT** [EDCD/EDMarketConnector#1256]
- Version: 5.1.3+ 1a0b4ae
 - Game Version: Live
 - OS: Windows 10
 - OS Locale: English
 - If applicable: Browser [e.g. chrome, safari]
 - Please attach **BOTH** log files, by dragging and dropping them into this input:
     
[EDMarketConnector-debug.log](https://github.com/EDCD/EDMarketConnector/files/7025518/EDMarketConnector-debug.log)

Is only generating this one log

**Describe the bug**
I stream Elite on the channel twitch.tv/malic_vr 

Because I play in VR, I try to offload everything to a secondary computer, so the gaming PC is not having to do any other work other then the rendering of the game.

I have recently been moving several programs over, and I got to ED Market Connector.  I have my journal folder shared so all of my computers on my network can access it.  What I discovered was that if one computer is already running EDMC, none of the other computers can start EDMC.  I plan to have the game PC running it to have it run the screenshot plugin, but the second computer is taking care of all other communication between the game and the EDDB for uploads of data

**To Reproduce**
1) share Elite journal folder on network
2) Install EDMC on both computers
3) Start EDCM on one computer
4) Attempt to launch EDMC on second computer (will not work)
5) shut down EDMC on first computer
6) start EDMC on second computer
7) start EDMC on first computer again (does nto work, because second computers copy has already hooked into the journal and it is not allowed to start)

**Additional context**
You might need to point each copy of the program to the journal (this applies mostly to the second computer because shared network location)

**Comment by Athanasius** [comment]
This is working as designed.

Especially in this shared folder case it would be very difficult for us to be sure it's *not* a local other copy of EDMarketConnector.exe holding the lock.

Keep in mind that double-sending to EDDN wastes resources, and double-sending to EDSM or Inara APIs would make a mess of your profile on them.

**Comment by MalicT** [comment]
I am not double sending, I still need EDMC on the game PC to do several of the other plugins that it has installed, all of its data sending is turned off, so the second computer handles those and deal with internet communication

**Comment by Athanasius** [comment]
I might be convinced to add a command-line argument to tell the program to specifically not care about that lock.  It won't be a GUI/Settings option, and anyone using it gets to limp if they shoot themself in the foot with it.

**Comment by A-UNDERSCORE-D** [comment]
> I might be convinced to add a command-line argument to tell the program to specifically not care about that lock. It won't be a GUI/Settings option, and anyone using it gets to limp if they shoot themself in the foot with it.

debug webservers forced on to eat any and all outgoing messages? possibly with a webserver that just blackholes requests?

**Comment by Athanasius** [comment]
> > I might be convinced to add a command-line argument to tell the program to specifically not care about that lock. It won't be a GUI/Settings option, and anyone using it gets to limp if they shoot themself in the foot with it.
> 
> debug webservers forced on to eat any and all outgoing messages? possibly with a webserver that just blackholes requests?

Yes, a possible use of our CL args for that if someone using the new CL arg for this issue is paranoid.

**Comment by Rixxan** [comment]
This will be included in the next release. Proceed with caution! :)
